cmds/installcommand/installcommand.go

```diff
@@ -171,16 +172,29 @@ func main() {
 	// and the glob will find it. If we get to the point that the glob
 	// no longer works we can go with filepath.Walk (which glob uses anyway)
 	// but for now this works.
-	src := filepath.Join("/src", util.CmdsPath, form.cmdName)
-	if _, err := os.Stat(src); err != nil {
-		l, err := filepath.Glob(filepath.Join("/src/*/*/", form.cmdName))
-		if err != nil || len(l) == 0 {
-			log.Fatalf("Can't find source code for %v (err %v)", form.cmdName, err)
+	var src string
+	for _, p := range util.CmdsGlob {
```

Comment:
Why not just filepath.Walk /src and match on any directory with the cmdName? Seems a little more robust than this glob.

Reply:
glob uses Walk, so from my POV, this is a lot less code that gets the same effect. I really prefer it.

Reply:
not the same effect -- if i were to add a command from say github.com/u-root/NiChrome/cmds/foo, that doesn't match the glob anymore, does it?

Reply:
btw, I'll take your word for it, we can fix this, but I'd like to get this PR in as is and have the discussion later. I think the glob is still more useful but maybe I'm totally wrong :-)